README.md

```diff
@@ -413,6 +384,36 @@ With this rule set, the following rules will apply:
 
 There are filters in place that can be applied to change the behavior for what's allowed and what's not allowed.
 
+### `vip_governance__governance_file_path`
+
+Change the governance rules file that's used by the plugin, based on a variety of filter options that are available. By default, it is set to
+the path to `governance-rules.json` in the private directory in a VIP site. For non-vip sites, it is set to the path to `governance-rules.json` in the plugin directory.
+
+```php
+/**
+ * Filter the governance file path, based on the filter options provided.
+ * 
+ * Currently supported keys:
+ * 
+ * site_id: The site ID for the current site.
```

Comment:
I wonder what else we might include to start. post_type and author are already covered by the rules. Are there other obvious values we could include? My first thought was publish status, but I don't think that's a very good one.

If we've all thought for 5-minutes and can't come up with something obvious, we should stop trying.

Reply:
I can't think of anything that the user isn't able to access themselves if they'd like. For example, even if we were missing `site_id`, user code can call into `get_current_blog_id()` directly instead, and pull other global values (like `$post`) out of the air.

It's still useful to include `$side_id` and possibly other parameters for convenience, but it hopefully won't be a blocker if customer code needs to use something else.

Reply:
I can't think of anything else either, so would leave it as just `site_id` for now. The upside of it being a key value map is that we can add to it, without any worries for backwards compatibility.